codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -809,6 +813,7 @@ private void lastKnownStream(int lastKnownStream) {
             if (!alreadyNotified) {
                 notifyGoingAway();
             }
+            closeStreamsGreaterThanLastKnownStream();
```

Comment:
@nmittler okay. do you think this is the correct place in the code to add this to?